Clear Project Guidelines Issue Resolution Try 3

Computer8543 opened the pull request:
I based this branch on a new fork of the central repository, and the changes are atomic this time. I also fixed the link in the README.md to the LICENSE.md so that it is relative instead of hardcoded, and it doesn't assume the name of the branch it is in. In my view, this push request should be ready to merge. If it is not, please let me know so we can work on this further.

michael-basil reviewed (commented):
This is a great reflection!   I really liked the points about human connection.  This is key.

I have verified all the criteria except the demo.

Looking forward to unpacking this further in the demo at Dojo Circle!

🙏🌿

billwestfall reviewed (approved):
Allison - great conversation and reflection!  
Bicycle metaphor - very interesting viewpoint on the well liked bicycle metaphor. Allison shared her own personal experience with this metaphor, leading us into a group discussion on this topic. 
Duck, rabbit - Allison had a great individual take on this item as well. Allison effectively challenged this learning item and Jeremy helped merge these topics together for discussion.
Allison's belt request also spurred a conversation on the topic of hierarchy and rank in the Dojo.
Allison's written reflections in the belt request are well defined and communicated as well. The struggle points were well defined and explained, and can be great topics for the circle as a whole coming up.
Allison's reflection on Zen at the gym is a great "look ahead" to Zen Flips and application of the principles of the Dojo for her red belt journey. 
Well done and congratulations Allison!

Computer8543 committed:
Add LICENSE.md

Computer8543 committed:
Add README.md

Computer8543 committed:
Update README.md

Changed hardcoded link to future central repository LICENSE.md to relative Markdown link for flexibility's sake.

Computer8543 committed:
Update README.md

Had to fix LICENSE.md link to not assume that the branch is named main.
